### ISSUE TITLE
defining number of people with a disease

### ISSUE THREAD

**Issue by nonieA** [synthetichealth/synthea#609]
Hey, 

From my experience currently when running synthea with a specific disease module, say COPD, you define the population and it returns that population number with x% having the disease. Is there a way of defining the number of patients with the disease rather than just the population number?

**Comment by jawalonoski** [comment]
No, there is currently no way of saying _"give me 10 patients all with COPD."_

The reason why is because of how we defined module transition types. The transition types are documented here: https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-Transitions

The transitions can be direct (i.e. fixed), or based on conditional logic, or distributions, or both conditional logic and distributions (aka complex).

In particular, the distributed transitions are always based on probability or percentages (e.g. _10% of the patients will have X and the remainder Y_), and never on a quantity (e.g. _10 total patients will have X_).

So, you can either generate a large population and then run a query for on patients with COPD and limit it to 10 (for example), or you can modify the module so everyone has COPD.

You can modify modules with the module builder: https://synthetichealth.github.io/module-builder/

**Comment by RachelElli** [comment]
Hi, @nonieA were you able to figure out a solution that worked for you?  I'm trying to do something similar with the dialysis module.

**Comment by nonieA** [comment]
Yeah I ended up writing a function in python that basically keeps running synthea until I have the desired number of patients with the disease, its not the most elegant of solutions but it does the job
